.travis.yml

```diff
@@ -2,6 +2,11 @@ language: go
 go:
 - 1.8.5
```

Comment:
Uer 1.8.x and 1.9.x or specify explicit release that is more recent. For 1.9 we are at 1.9.4 at the moment.

Reply:
Also, you may want to update your Dockerfile's to reference 1.9.4 (or whatever latest you would want it to be built with).

Reply:
I'll move them all to 1.9.4, and remove 1.8.5.   Although, there is 1.10...